platform/src/main/java/org/hillview/table/api/IColumn.java

```diff
@@ -118,7 +118,12 @@ default String getName() {
     default ContentsKind getKind() { return this.getDescription().kind;}
 
     /**
-     * @return A hash code for the item in the rowIndex. Exception if it is missing.
+     * @return A 64 bit hash code for the item in the rowIndex.  xors the item with the seed and then runs xxhashing
```

Comment:
I have another idea: how about replacing the seed with a reference to the XXHash?
The prototype is
```
long hashCode64(int rowIndex, XXHash hash);
```